docs/PUBLISHING.md

```diff
@@ -4,6 +4,27 @@ Publishing Etcher
 This is a small guide to package and publish Etcher to all supported operating
 systems.
 
+Release Types
+-------------
+
+Etcher supports **production** and **snapshot** releases. Each is published to
+a different S3 bucket, and production releases are code signed, while snapshot
+releases aren't and include a short git commit-hash as a build number. For
+example, `1.0.0-beta.19` is a production release, while `1.0.0-beta.19+531ab82`
+is a snapshot release.
+
+In terms of comparison: `1.0.0-beta.19` (production) < `1.0.0-beta.19+531ab82`
+(snapshot) < `1.0.0` (production) < `1.0.0+2201e5f` (snapshot).
+
+The build system creates snapshot releases by default, but you can build a
+specific release type by setting the `RELEASE_TYPE` environment variable. For
+example:
+
+```sh
+RELEASE_TYPE=snapshot
```

Comment:
My intention was that you can pass `RELEASE_TYPE=<type>` to `make`, so that is platform independent. I clarified the example.